### ISSUE TITLE
[Request] Do not keep messages containing IP addresses or other doxing material after message deletion

### ISSUE THREAD

**Issue by jasonwynn10** [SevenTV/SevenTV#263]
Doxing has become more prevalent than ever with hate raids and distasteful interactions on Twitch. Recently I witnessed an issue where a streamer had their IP, home address, and latitude/longitude leaked in a chat message. It was deleted rather quickly by an active moderator, but for users using chat extensions the message stayed in full view.

The current system of keeping messages shown is extremely useful, but it's also given that doxxers use it to their advantage. I'm requesting a change to stop showing IPs specifically because it's not as complicated to set up regex to check for it compared to other more complicated means.

TLDR; If messages are deleted and the message contains information that breaches someone's privacy (namely IPs), the message should be deleted.

**Comment by AnatoleAM** [comment]
While a setting to manage the visibility of deleted messages could and eventually will be added, the concept of an extension hiding content arbitrarily would raise eyebrows as that is completely out of scope.

There are many other extensions and apps that allow viewing deleted content (including the official twitch mobile app), so this is not a proper solution; once a message is sent, it cannot be unsent, and no amount of client-side alterations will thwart a malicious actor. Ultimately, user safety is Twitch's responsibility, but there is no one size fits all fix to this issue.
